portfolio/src/main/webapp/script.js

```diff
@@ -38,3 +38,12 @@ function goBack() {
         history.back();
     }
 }
+
+/**
+ * Fetches the response of "/data".
+ */
+function getData() {
```

Comment:
Nit: maybe rename to loadAndShowData()? That's what this method do.
Optionally can also refactor implementation into two methods loadData().then((data) => { showData(data) }

Reply:
Fixed.